lib/kamal/commands/builder/base.rb

```diff
@@ -97,6 +97,10 @@ def build_ssh
       argumentize "--ssh", ssh if ssh.present?
     end
 
+    def builder_provenance
+      argumentize "--provenance", provenance unless provenance.nil?
```

Comment:
```suggestion
      argumentize "--provenance", provenance if provenance.present?
```

Reply:
This is intentional design, because if `provenance` is `false`, it should output the `--provenance false` option. It should only skip outputting `--provenance …` if `provenance` is nil.